salt/orch/update.sls

```diff
@@ -57,9 +57,10 @@ update_modules:
   salt.function:
     - tgt: {{ master_id }}
     - name: cmd.run
-    - bg: True
     - arg:
-      - systemctl reboot
+      - sleep 15; systemctl reboot
```

Comment:
If I remember correctly from @grahamhayes - this didn't actually work from orchestration, but it's been a while.